WeatherApp/Data/PersistenceManager.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  PersistenceManager.swift
+//  WeatherApp
+//
+//  Created by Steven on 7/24/19.
+//  Copyright © 2019 Project Yato. All rights reserved.
+//
+
+import Foundation
+
+enum PersistencePath: String {
+    case IsMetric = "IsMetric"
+    case Location = "Location"
+    case LatLong = "LatLong"
+}
+
+class PersistenceManager {
+    class fileprivate func documentsDirectory() -> URL {
+        let paths = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)
+        return paths[0]
+    }
+    
+    class func save(_ save: AnyObject, path: PersistencePath) {
+        let filePath = documentsDirectory().appendingPathComponent(path.rawValue)
+        
+        do {
+            let data = try NSKeyedArchiver.archivedData(withRootObject: save, requiringSecureCoding: false)
+            try data.write(to: filePath)
+            print("Success saving data")
+            LogManager.instance.Log.trace("Success saving data")
+        } catch let err {
+            LogManager.instance.Log.error(err, terminator: "Unable to save data")
+        }
+    }
```

Comment:
Nice, we're using some of this in the app right now. Going to transition to CoreData soon but we'll always need to archive to file or UserDefaults.